openupgradelib/openupgrade_merge_records.py

```diff
@@ -455,7 +455,13 @@ def merge_records(env, model_name, record_ids, target_record_id,
     args0 = (env, model_name, record_ids, target_record_id)
     args = args0 + (exclude_columns, )
     args2 = args0 + (field_spec, )
-
+    if target_record_id in record_ids:
+        raise Exception("You can't put the target record in the list or "
```

Comment:
.ValidationError,.UserError? :S

Reply:
It should be a general Exception, as this is a library, not ORM code.

Reply:
ok